### PR TITLE
Updates to README.md

### PR DESCRIPTION
**Pull request by DukeDavis12**
Updating the AIO README files to include the new OnPrem-RV
   integration information and to reflect the changes in this release.

Signed-off-by: Davis Benny <davis.benny@intel.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # About
 
 The Secure Device Onboard All-In-One demo is designed to run different Secure Device Onboard
-services like Manufacturer, IoT Platform SDK Service and Rendezvous Service within a single
-instance of tomcat server. The purpose of this demo is to fast-track demonstration of Secure
+services like Manufacturer, IoT Platform SDK Service and Rendezvous Service from a single
+**docker-compose** instance. The purpose of this demo is to fast-track demonstration of Secure
```
Comment (trbehera) [inline README.md:5]: docker-compose => docker

File: README.md [hunk 2]
```diff
@@ -41,14 +41,23 @@ The All-In-One demo is dependent on following repositories.
...
 
-After successful build, the demo package is available at container/target/aio.tar.gz.
+Alternatively, you have to option to build All-in-One demo using Docker.[Read more](build/README.md)
+
+```
+cd build/
+docker-compose up --build
+```
+
+After successful build, the demo package is available at container/demo/aio.tar.gz.
```
Comment (trbehera) [inline README.md:60]: container/demo/aio.tar.gz => demo/aio.tar.gz

File: container/overlay/README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # About
 
 The Secure Device Onboard (SDO) All-In-One demo is designed to run SDO Manufacturer service, SDO IoT
-Platform SDK Service and SDO Rendezvous Service within a single instance of tomcat server. The
+Platform SDK Service and SDO Rendezvous Service from a single **docker-compose** instance. The
```
Comment (trbehera) [inline container/overlay/README.md:4]: docker-compose => docker

**Review by trbehera — CHANGES_REQUESTED** [review]

File: container/overlay/README.md (hunk 3 above)
Comment (DukeDavis12) [reply container/overlay/README.md]: Updated.

File: README.md (hunk 2 above)
Comment (DukeDavis12) [reply README.md]: Updated.

File: README.md (hunk 1 above)
Comment (DukeDavis12) [reply README.md]: Updated.

**Review by trbehera — APPROVED** [review]

**Review by KiranSukhavasi — APPROVED** [review]

**Review by trbehera — APPROVED** [review]

**Review by KiranSukhavasi — APPROVED** [review]

**Pull request merged**